test/integration/model/bulk-create.test.js

```diff
@@ -445,11 +446,15 @@ describe(Support.getTestDialectTeaser('Model'), () => {
         await this.User.bulkCreate(data, { fields: ['uniqueName', 'secretValue'] });
         data.push({ uniqueName: 'Michael', secretValue: '26' });
 
+        let thrown = false;
         try {
           await this.User.bulkCreate(data, { fields: ['uniqueName', 'secretValue'], ignoreDuplicates: true });
         } catch (error) {
+          thrown = true;
           expect(error.message).to.equal(`${dialect} does not support the ignoreDuplicates option.`);
         }
+
+        expect(thrown).to.equal(true);
```

Comment:
`.to.be.rejectedWith` is the recommended way to test that an async method throws:

```suggestion
        await expect(this.User.bulkCreate(data, { fields: ['uniqueName', 'secretValue'], ignoreDuplicates: true })).to.be.rejectedWith(`${dialect} does not support the ignoreDuplicates option.`);
```